docs: add information about stream service to service (#1495)
encoredev/encore: docs/ts/primitives/streaming-apis.mdx

});

```

## Service to service streaming

> Like with any [other endpoints](/docs/ts/primitives/api-calls) you can easily use streaming between services by importing `~encore/clients`.

Reviewer: ```suggestion
Like with [other endpoint types](/docs/ts/primitives/api-calls) you can easily use streaming between services by importing `~encore/clients`.
```

> If you want the stream to only be reachable by other services, set the `exposed`-option to false.

Reviewer: ```suggestion
If you want the stream to only be reachable by other services (and not from the public internet), set the `exposed` option to false.
```

Example of using a stream endpoint from a regular api endpoint:

```typescript

> import { streaming } from "~encore/clients"; // import 'streaming' service

Reviewer: ```suggestion
import { chat } from "~encore/clients"; // import 'chat' service
```

export const myOtherAPI = api({}, async (): Promise<void> => {

>   const stream = await streaming.streamEndpoint();

Reviewer: ```suggestion
  const stream = await chat.myStreamingEndpoint();
```

>   // send a message to the other service

Reviewer: ```suggestion
  // send a message to the chat service over the stream
```
